InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -25,6 +27,9 @@
     int targetPosition;
     int targetChange;
 
+    // Class Declarations
+    RunningAverageFilter filter = new RunningAverageFilter(25);

```

Comment:
looks good, but we may want to put 25 in a ShooterConstants class to make it more easily accessible. Typically we don't hard code these "magic numbers" into the subsystems/commands and rather have them access the constant from another class dedicated to storing constants. Take a look at the DriveConstants (frc.robot.subsystem.drive) - use the same "public static final <variableType> (in this case, int because 25 is an integer)> <VARIABLE_NAME> = <WHATEVER>" and then from outside you can access it as ShooterConstants.<VARIABLE_NAME>